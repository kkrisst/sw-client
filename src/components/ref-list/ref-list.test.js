import React from 'react'
import { shallow } from 'enzyme'

import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'

import RefList from './ref-list.jsx'

describe('Testing the RefList component', () => {

  const initialState = {};
  const mockStore = configureStore();


  it('expect to render RefList component', () => {
    const store = mockStore(initialState);
    expect(shallow(<Provider store={store}><RefList /></Provider>)).toMatchSnapshot()
  })

  it('expect to render RefList component', () => {
    const store = mockStore(initialState);
    const wrapper = shallow(<Provider store={store}><RefList refs={['a', 'b', 'c']} /></Provider>);
    const linkElement = wrapper.find('.ref-list')
    expect(linkElement.length).toEqual(3)
  })

})