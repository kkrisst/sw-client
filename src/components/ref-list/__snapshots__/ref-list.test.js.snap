// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the RefList component expect to render RefList component 1`] = `ShallowWrapper {}`;
